### PR TITLE
SA-2212 - (UI) Re-organize existing integration tests for Analytics Report

### PR DESCRIPTION
**Pull request by nicklemmon**
[SA-2212](https://sparkpost.atlassian.net/browse/SA-2212)

### What Changed

- Moved a test from `reportPage.spec.js` to `metricsForm.spec.js`
- Renamed a few test cases
- Added a missing assertion or two while I was there
- Renamed most test suites for the sake of consistency

### How To Test

- Ensure tests were moved to the correct location
- Any other cleanup that could be done while we're here?

### To Do

- [ ] Incorporate feedback

### CODE REVIEW

File: cypress/tests/integration/signals-analytics/analytics-report/metricsForm.spec.js [hunk 1]
```diff
@@ -1,7 +1,67 @@
-import { PAGE_URL, INBOX_METRICS } from './constants';
+import { PAGE_URL, INBOX_METRICS, METRICS } from './constants';
 import { commonBeforeSteps } from './helpers';
 
-describe('Metrics form', () => {
+describe('Analytics Report metrics form', () => {
+  it("adds active metrics according to the user's entry", () => {
```
Comment (nicklemmon) [inline cypress/tests/integration/signals-analytics/analytics-report/metricsForm.spec.js:5]: Moved this from `reportPage.spec.js` and renamed it.

File: cypress/tests/integration/signals-analytics/analytics-report/metricsForm.spec.js [hunk 2]
```diff
@@ -58,6 +118,50 @@ describe('Metrics form', () => {
       });
     });
 
+    it('should properly control the expandables', () => {
```
Comment (nicklemmon) [inline cypress/tests/integration/signals-analytics/analytics-report/metricsForm.spec.js:121]: I moved this one as well - and added a missing expandable that wasn't tested.

File: cypress/tests/integration/signals-analytics/analytics-report/reportPage.spec.js [hunk 3]
```diff
@@ -10,7 +10,7 @@ import {
 const ENCODED_QUERY_FILTERS =
   '%255B%257B%2522AND%2522%3A%257B%2522campaigns%2522%3A%257B%2522like%2522%3A%255B%2522hello%2522%2C%2522world%2522%255D%257D%2C%2522templates%2522%3A%257B%2522eq%2522%3A%255B%2522greg-hackathon%2522%255D%2C%2522notEq%2522%3A%255B%2522gregs-test%2522%255D%257D%257D%257D%2C%257B%2522AND%2522%3A%257B%2522sending_ips%2522%3A%257B%2522like%2522%3A%255B%2522hello%2522%255D%2C%2522notLike%2522%3A%255B%2522hello-again%2522%255D%257D%257D%257D%255D';
 
-describe('Analytics Report', () => {
+describe('The Analytics Report page', () => {
```
Comment (nicklemmon) [inline cypress/tests/integration/signals-analytics/analytics-report/reportPage.spec.js:13]: Thought this was more specific?

**Review by PatrickSison — APPROVED** [review]
These test moves look good.  Also good changes to the test

**Pull request merged**